components/brave_shields/browser/brave_farbling_service.cc

```diff
@@ -60,8 +62,10 @@ bool BraveFarblingService::MakePseudoRandomGeneratorForURL(
 // static
 void BraveFarblingService::RegisterProfilePrefs(
     user_prefs::PrefRegistrySyncable* registry) {
-  registry->RegisterBooleanPref(brave_shields::prefs::kReduceLanguageEnabled,
-                                true);
+  registry->RegisterBooleanPref(
+      brave_shields::prefs::kReduceLanguageEnabled,
+      base::FeatureList::IsEnabled(
```

Comment:
@bbondy said this change was necessary to make it work, but that means there is another problem because it should not be necessary. If someone has a non-default value for the pref (already toggled it off and on again), disabling the feature won't actually disable it.

Reply:
somewhere in the code we are missing a feature guard

Reply:
sounds like maybe the issue here is just a UI problem because the toggle shouldn't appear at all if the feature is disabled